src/app/data-model-provider/OperationTypes.h

```diff
@@ -70,15 +70,24 @@ struct ReadAttributeRequest : OperationRequest
 
 enum class WriteFlags : uint32_t
 {
-    kTimed     = 0x0001, // Write is a timed write (i.e. a Timed Request Action preceeded it)
-    kListBegin = 0x0002, // This is the FIRST list of data elements
-    kListEnd   = 0x0004, // This is the LAST list element to write
+    kTimed = 0x0001, // Write is a timed write (i.e. a Timed Request Action preceeded it)
 };
 
 struct WriteAttributeRequest : OperationRequest
 {
     ConcreteDataAttributePath path; // NOTE: this also contains LIST operation options (i.e. "data" path type)
     BitFlags<WriteFlags> writeFlags;
+
+    // The path of the previous successful write in in the same write transaction
+    //
+    // In particular this means that a write to this path has succeeded before (i.e. it passed required ACL checks).
+    // The intent for this is to allow short-cutting ACL checks when ACL is in progress of being updated:
+    //   - During write chunking, list writes are of the form "clear list" followed by "append item by item"
```

Comment:
```suggestion
    //   - During write chunking, list writes can be of the form "reset list" followed by "append item by item"
```
The initial reset might not be to empty list....